modules/new_profile/jsx/NewProfileIndex.js

```diff
@@ -89,6 +90,9 @@ class NewProfileIndex extends React.Component {
       }
       formObject.append('fire_away', 'New Candidate');
 
+      // disable form from resubmission.
+      this.setState({disable: true});
```

Comment:
I would call this something a bit more descriptive like `disableSubmit` or something to that affect. Perhaps `submitDisabled` because it is written as more a state (true/false) whereas `disable` sounds like an action (e.g. function).

Reply:
Also, I think if the submission fails, the button should be re-enabled so that the user can attempt another submission.

Reply:
Thanks @HenriRabalais, I added the suggested changes.